### PR TITLE
ES-906129 - Modify Prsentation GitHub examples repository for document processing playground 

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Completed Git hub sample changes for he below files
Open and save
PPT to Image
Getting Started

### CODE REVIEW

File: Getting-started/Create-PowerPoint-with-basic-elements/.NET/Create-PowerPoint-with-basic-elements/Program.cs [hunk 1]
```diff
@@ -37,11 +37,11 @@
 paragraph.IndentLevelNumber = 2;
 
 //Get a picture as stream.
-using FileStream pictureStream = new(@"../../../Data/Image.jpg", FileMode.Open);
+using FileStream pictureStream = new(@"Data/Image.jpg", FileMode.Open);
```
Comment (LokeshBaskar) [inline Getting-started/Create-PowerPoint-with-basic-elements/.NET/Create-PowerPoint-with-basic-elements/Program.cs:40]: Please use the Path.GetFullPath()

File: PPTX-to-Image-conversion/Add-font-substitution/.NET/Add-font-substitution/Program.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ internal class Program
         static void Main()
         {
             //Load the PowerPoint presentation and convert to image
-            using (IPresentation pptxDoc = Presentation.Open(@"../../../Data/Template.pptx"))
+            using (IPresentation pptxDoc = Presentation.Open(@"Data/Template.pptx"))
```
Comment (LokeshBaskar) [inline PPTX-to-Image-conversion/Add-font-substitution/.NET/Add-font-substitution/Program.cs:11]: Please use Path.GetFullPath

File: PPTX-to-Image-conversion/Add-font-substitution/.NET/Add-font-substitution/Program.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@ static void Main()
                 using (Stream stream = pptxDoc.Slides[0].ConvertToImage(ExportImageFormat.Jpeg))
                 {
                     //Create the output image file stream
-                    using (FileStream fileStreamOutput = File.Create("Output.jpg"))
+                    using (FileStream fileStreamOutput = File.Create("Output/Output.jpg"))
```
Comment (LokeshBaskar) [inline PPTX-to-Image-conversion/Add-font-substitution/.NET/Add-font-substitution/Program.cs:21]: Please use Path.GetFullPath

File: PPTX-to-Image-conversion/Based-on-Entrance-animation-effects/.NET/Based-on-Entrance-animation-effects/Program.cs [hunk 4]
```diff
@@ -15,7 +15,7 @@ internal class Program
         static void Main(string[] args)
         {
             //Opens a PowerPoint Presentation
-            IPresentation pptxDoc = Presentation.Open(@"../../../Data/AnimationConverter.pptx");
+            IPresentation pptxDoc = Presentation.Open(@"Data/AnimationConverter.pptx");
```
Comment (LokeshBaskar) [inline PPTX-to-Image-conversion/Based-on-Entrance-animation-effects/.NET/Based-on-Entrance-animation-effects/Program.cs:18]: Please use Path.GetFullPath

File: PPTX-to-Image-conversion/Based-on-Entrance-animation-effects/.NET/Based-on-Entrance-animation-effects/Program.cs [hunk 5]
```diff
@@ -34,7 +34,7 @@ static void Main(string[] args)
                         stream.Position = 0;
 
                         //Create the output image file stream.
-                        using (FileStream fileStreamOutput = File.Create(@"../../../Output" + i + ".png"))
+                        using (FileStream fileStreamOutput = File.Create(@"Output/Output" + i + ".png"))
```
Comment (LokeshBaskar) [inline PPTX-to-Image-conversion/Based-on-Entrance-animation-effects/.NET/Based-on-Entrance-animation-effects/Program.cs:37]: Please use Path.GetFullPath

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/.NET/Convert-PowerPoint-presentation-to-Image/Program.cs [hunk 6]
```diff
@@ -22,7 +22,7 @@ static void Main(string[] args)
                     foreach (Stream stream in images)
                     {
                         //Create the output image file stream
-                        using (FileStream fileStreamOutput = File.Create("Output" + Guid.NewGuid().ToString() + ".jpg"))
+                        using (FileStream fileStreamOutput = File.Create("Output/Output" + Guid.NewGuid().ToString() + ".jpg"))
```
Comment (LokeshBaskar) [inline PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/.NET/Convert-PowerPoint-presentation-to-Image/Program.cs:25]: Please check and use the Path.GetFullPath while open and save document

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]
Please address the below feedbacks

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**